### ISSUE TITLE
YubiKey not supported

### ISSUE THREAD

**Issue by ziifctp2** [robb-j/MiniYubiOath#1]
I'm using a YubiKey 5C NFC with a password. Yoath tells me that my YubiKey is not supported.

Are encrypted YubiKeys not supported?

**Comment by robb-j** [comment]
Hey, thanks for trying/using the app. I couldn’t get it working before and don’t use it myself so it’s currently not supported. I could definitely take another stab at it if you like? Is there a password per credential/code or a single password for the entire OATH module (I’m not entirely sure how it works)?

**Comment by ziifctp2** [comment]
Hey! I've set a password for the whole module.

**Comment by dalishi188** [comment]
Yes, it cannot be used. There is no way. I look forward to your hard work.

**Comment by michael-kallweitt** [comment]
I'd also like to see the YubiKey 5C supported.
